### DISCUSSION TITLE
Alerts sound continuously

### DISCUSSION THREAD

**Discussion by thederekFawcett** [NightscoutFoundation/xDrip#3338] [Bug reports]
Discussion for **Issue #1418 **:
[https://github.com/NightscoutFoundation/xDrip/issues/1418](url)

Version: 3e86af7-2023.11.14
Code: 2123111417
Samsung S21

When any "Alarms and Alerts" are set (Glucose Level Alert, Other Alerts, Extra Alerts), when first triggered the phone will trigger the alert beep/sound at full volume, every minute until dismissed.

- "re-raise every x minutes if unacknowledged" values do not affect outcome
- "start snoozed" does work and you can dismiss it. If you miss it, or the option is unselected, then you get the once per minute sounded alarm
- "alert volume profile" set to "ascending volume" will still start at max volume and continue the alarm at max volume.

I guess the "re-raise" function and "ascending volume" are both at fault, two separate problems. This has been an issue since mid-2021, which is when we started using alerts.

![304265840-049758c3-c26e-4d47-b480-d770ad4a0deb](https://github.com/NightscoutFoundation/xDrip/assets/11082623/fb455a97-2d22-4ddf-a359-1659515510a5)
![304267010-66864e33-5dbf-4768-9e41-d3ab75cf7d35](https://github.com/NightscoutFoundation/xDrip/assets/11082623/b4d1200e-80d1-4e4a-99bf-4e072f531d9f)
![304267019-8ada9190-507f-426f-933c-5d893186d512](https://github.com/NightscoutFoundation/xDrip/assets/11082623/cc17b2d9-6139-435a-9a43-3542b9152d21)

**Comment by Navid200** [comment]
You have enabled start snoozed.  Why?

**Reply by thederekFawcett** [reply]
To have the ability to dismiss the alert before the once-per-minute alerts start, in case we get distracted and aren't near the device when it sounds.

**Reply by Navid200** [reply]
If you choose the ascending volume profile, your glucose level alerts will sound every minute.  This has nothing to do with reraise.

You can dismiss the alert any time regardless.

**Reply by thederekFawcett** [reply]
testing...
There should be that exact note next to the "Ascending Volume" option, otherwise our intuition understands it as each raise of the alert will increase in volume. But its intention is to start at low volume and increase during the ring of the one alert, like a phone ring or wake-up alarm.

I guess in the Dexcom world, all alerts function like a short notification beep/chime, not a long notification like a call or alarm.

**Reply by Navid200** [reply]
There is no way we can place all the details inside xDrip.  That's why I have been maintaining a guide.  This page of it is just about the Ascending volume profile:
https://navid200.github.io/xDrip/docs/Ascending-volume-profile.html

There is so much in xDrip.  I have been trying to clean up the user interface.  It is not easy.  Many of my submissions are held up because we have to be careful how it may impact someone who has got used to the old ways.

**Reply by thederekFawcett** [reply]
> Every time it plays, the volume increases a little bit. Eventually, it plays it at the max. volume.

So then this does not work. It will start at max and continue at max every minute until dismissed.

**Reply by Navid200** [reply]
Disable start snoozed and see if it happens again.  I suspect it won't.
I have been using xDrip for 6 years now and have been using the Ascending volume profile all along.  It never starts at max level.
But, I have never enabled start snoozed.  I wish that setting had never been added.  Frankly, I don't know why it is there.

**Comment by thederekFawcett** [comment]
> If you choose the ascending volume profile, your glucose level alerts will sound every minute. This has nothing to do with reraise.
> 
> You can dismiss the alert any time regardless.

When set to not Ascending Volume, a single "beep" type notification will keep beeping continuously I'm assuming forever until dismissed. When set to Ascending Volume, the single beep will play once at max volume every minute.

**Reply by Navid200** [reply]
You have chosen to buy a Samsung.  You have to put up with it.
Samsung lets you customize all notifications.  Search on Google and find out how it lets you customize them.  It is very likely you have a customization that causes that.

**Reply by thederekFawcett** [reply]
No custom notifications or settings involved. The device only has Dexcom and xDrip installed outside of the standard issue Google & Samsung ones that can't be uninstalled. It's only purpose is a diabetes device. You've stated in other issues and discussions that the "Alert Volume Profile" is independent of device volume set. Plus it's most popular Android brand on the planet and has the broadest range of compatibility with medical devices.

**Reply by Navid200** [reply]
There is nothing in xDrip that causes a beep for an alert.
xDrip heavily uses notifications for alerts.  
You have enabled vibrate on alert.  It is very likely that your phone beeps when it vibrates.  That is a phone setting.  You must be able to disable that.

**Reply by thederekFawcett** [reply]
We've never used "Vibrate" as a "Alert Volume Profile" notification option because the device is almost never held on person. The "beep" I type is used in commonplace for a sound alert notification. The standard issue sound is too jarring for non-critical notifications like one set for a just above average blood sugar, so we use device OEM sounds.

**Reply by thederekFawcett** [reply]
setting up notifications now on a Moto now to test to appease you

**Reply by Navid200** [reply]
According to your screenshot, you have enabled vibrate on alert.

**Reply by thederekFawcett** [reply]
Yes the phone vibrates when the set audible notification is triggered to aid in sensing an alert in a loud environment. When set to Medium, High, or Ascending Volume, only one sound file plays.

**Reply by Navid200** [reply]
I thought you said you never used vibrate.

I don't have your phone.  I am looking at your screenshots and see that you have enabled vibrate.  I know that when ascending volume is used, it vibrates every minute if you have vibrate on alert enabled.  
And you say that it beeps every minute with ascending volume, but only beeps once with medium or high.   So, I put 2 and 2 together and it is very likely that you have a setting that beeps for every vibration.

I am trying to help you.  It really makes no difference if I prove you wrong.  I will get no raise.  We are all volunteers.  If in the process we find out that something is wrong with xDrip, we will fix it.

Can you put yourself in my shoes?  I have no way of experiencing what you describe.  The only thing I have is what you post.
You said you never used vibration.   Now, you say you do.

**Reply by Navid200** [reply]
OK, I see you meant you have not used the vibrate volume profile.  You don't have to for vibrate on alert to vibrate.

**Reply by Navid200** [reply]
Disable this and see what happens with the beep.  
![304441323-b4d1200e-80d1-4e4a-99bf-4e072f531d9f](https://github.com/NightscoutFoundation/xDrip/assets/51497406/4b9ba5f5-aa98-4948-9f4b-f749dc97b3bb)

**Reply by thederekFawcett** [reply]
[https://imgur.com/a/gPE5T2m](Imgur)
_links are stupid and I give up, just copy & paste_
xDrip on Moto. Same occurrence happens on every device. 
So "Ascending Volume"  negates "re-raise every x minutes"?

**Reply by Navid200** [reply]
I have a few Moto phones.  I have never experienced this.  I cannot access the link you have posted.

**Reply by thederekFawcett** [reply]
I guess in comments it won't let you link to an external URL
(https://imgur.com/a/gPE5T2m)

ooh this one worked
Video is one minute to show first alert, then another one minute later

**Reply by Navid200** [reply]
As a troubleshooting step, can you disable notification channels and see if it still beeps?
https://navid200.github.io/xDrip/docs/Notification-channels.html

**Reply by thederekFawcett** [reply]
Sorry, got busy. I'll try it later or tomorrow

**Reply by thederekFawcett** [reply]
> As a troubleshooting step, can you disable notification channels and see if it still beeps?
> https://navid200.github.io/xDrip/docs/Notification-channels.html

Sorry, busy. "Notification Channels" doesn't affect outcome.

**Reply by Navid200** [reply]
Is the video clip you have posted from a Samsung or Moto?
If Moto, what model is it?

**Reply by thederekFawcett** [reply]
Takes place on a Moto G Stylus 5G 2023. Behavior on Moto or Samsung, Android 12-14 (only two brands of Android I currently own).

**Reply by Navid200** [reply]
I have a Moto G Stylus.  I think it is 2022.
Please save all settings (not the database) into a file.  https://navid200.github.io/xDrip/docs/CopySettings.html

Then, send it to me:
xDrip dot Navid at gmail dot com

I would like to load it and see if it beeps for me too.  Then, I should be able to figure out why.

**Reply by thederekFawcett** [reply]
(https://drive.google.com/file/d/1EcKTaXpptj3U7od1VVoiKWOpP99ZB1r7/view?usp=drivesdk)

Won't let me upload that file here, so hopefully this Google Drive link works. It's from a Moto G Stylus 5G 2022

**Reply by Navid200** [reply]
This is just a troubleshooting step.  Would you please make the following change and tell me if it makes any difference?
Please disable the following setting:
https://navid200.github.io/xDrip/docs/AAPS_ProfileImportNotification.html

**Reply by thederekFawcett** [reply]
"Import Sounds" is checked but greyed out, unable to be changed

_Edit:_ I had to enable "Accept Treatments" to uncheck it, then disabled "Accept Treatments" again. I'll see what happens later today
